reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/Delay.kt

```diff
@@ -0,0 +1,39 @@
+package com.badoo.reaktive.completable
+
+import com.badoo.reaktive.base.subscribeSafe
+import com.badoo.reaktive.disposable.CompositeDisposable
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.scheduler.Scheduler
+
+fun Completable.delay(delayMillis: Long, scheduler: Scheduler, delayError: Boolean = false): Completable =
```

Comment:
Do you want to add default scheduler as RxJava do?

Reply:
Let's discuss. I find default scheduler as `computation` very error prone, as in most of the cases you want main thread. We had some crashes due to such mistakes. So I would prefer either to define default value as `mainScheduler` or to just keep it mandatory.
@CherryPerry @lukaville @mk-jones What do guys think?

Reply:
I agree that it can be error prone.

I disagree with setting it mainScheduler however. I think by breaking convention with RxJava would also lead to unexpected bugs for users. I think the choice should be between providing computation as default, or to have it provided explicitly.

I would personally prefer leaving it explicit.

Reply:
> as in most of the cases you want main thread
In most of our cases.
Other people do a lot of staff on threads other than main. I prefer to not have any scheduler rather than have main as default.

Reply:
Rxjava does not have main scheduler, but we do. Let's keep it explicit then.